Rakefile

```diff
@@ -51,7 +51,15 @@ def getAgentTrackingInfo
   end
 end
 
+# Packaging into a tar
+require 'rake/packagetask'
```

Comment:
Move `require` line to top of the Rakefile

Reply:
Moved